virttest/remote_commander/remote_runner.py

```diff
@@ -768,19 +791,31 @@ def remote_agent(in_stream_cls, out_stream_cls):
     :params out_stream_cls: Class encapsulated output stream.
     """
     try:
-        fd_stdout = sys.stdout.fileno()
-        fd_stderr = sys.stderr.fileno()
-        fd_stdin = sys.stdin.fileno()
+        sys.stdout.flush()
+        sys.stderr.flush()
+        fd_stdin = sys.stdin.fileno()    # == 0
+        fd_stdout = sys.stdout.fileno()  # == 1
+        fd_stderr = sys.stderr.fileno()  # == 2
+        orig_stdout = os.dup(fd_stdout)  # Original stdout to master.
+        orig_stderr = os.dup(fd_stderr)  # Original srderr to master.
```

Comment:
Yes.
All SSH encoded base64 traffic goes through stdout.
Executed program/script has modified stdout/stderr.
stderr/stdout from remote_runner are endcoded and passed through original SSH stdout.
So.. in general stdout from SSH encapsulates stdout/stderr from program.
What you have pointed is not a bug. We can safely remove this variable. But...
I am sure that this code is a bit "tricky", so I just left this variable to represent what is really going here.
I can remove it. But it is safe to keep it for demonstrativeness. Do you want me to remove this variable ?